script/lib/check-chromedriver-version.js

```diff
@@ -22,24 +26,15 @@ module.exports = function() {
     );
   }
 
-  const electronVer = CONFIG.appMetadata.electronVersion;
-  if (!semver.satisfies(electronVer, chromedriverVer)) {
+  if (!semver.satisfies(chromedriverActualVer, '>=9.0.0')) {
     throw new Error(
-      `electron-chromedriver ${chromedriverVer} incompatible with electron ${electronVer}.\n` +
-        'Did you upgrade electron in package.json and forget to upgrade electron-chromedriver in ' +
-        `script/package.json to '~${semver.major(electronVer)}.${semver.minor(
-          electronVer
-        )}' ?`
+      `electron-chromedriver should be at least v9.0.0 to support the ELECTRON_CUSTOM_VERSION environment variable.`
     );
   }
 
-  if (!semver.satisfies(electronVer, mksnapshotVer)) {
+  if (!semver.satisfies(mksnapshotActualVer, '>=9.0.2')) {
     throw new Error(
-      `electron-mksnapshot ${mksnapshotVer} incompatible with electron ${electronVer}.\n` +
-        'Did you upgrade electron in package.json and forget to upgrade electron-mksnapshot in ' +
-        `script/package.json to '~${semver.major(electronVer)}.${semver.minor(
-          electronVer
-        )}' ?`
+      `electron-chromedriver should be at least v9.0.2 to support the ELECTRON_CUSTOM_VERSION environment variable.`
```

Comment:
I think it should be `electron-mksnapshot` instead of `electron-chromedriver`

Reply:
That's right, thanks for catching that.